src/components/LayoutHeader/DocSearch.js

```diff
@@ -25,6 +25,10 @@ class DocSearch extends Component<{}, State> {
         apiKey: '36221914cce388c46d0420343e0bb32e',
         indexName: 'react',
         inputSelector: '#algolia-doc-search',
+        handleSelected: function(input, event, suggestion, datasetNumber, context) {
+          const suggestionUrl = new URL(suggestion.url)
+          window.location = `${window.origin}${suggestionUrl.pathname}`
```

Comment:
All paths are coming with a slash from Algolia, that's why I haven't added slash between `window.origin` and `suggestionUrl.pathname`

More info about `handleSelected` behavior:
https://docsearch.algolia.com/docs/behavior/